hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -49,6 +49,9 @@
   @Parameter(names = {"--jdbc-url"}, description = "Hive jdbc connect url", required = true)
   public String jdbcUrl;
 
+  @Parameter(names = {"--metastore-uri"}, description = "Hive metastore uri)")
+  public String hiveMetaStoreUri;
+
   @Parameter(names = {"--base-path"}, description = "Basepath of hoodie table to sync", required = true)
```

Comment:
pleasse improve the rescription that this paramter is required when sync data to remote hive

Reply:
BTW, can you make `HoodieDeltaStreamer` and `SparkDataSource` support this feature too ?

Reply:
good idea

Reply:
@Trevor-zhang Did you address @wangxianghu 's concern?

Reply:
I have adjusted in [https://issues.apache.org/jira/browse/HUDI-1530](HDUI-1530) and will submit this pr later . Cc @yanghua